Kitodo/src/main/java/org/kitodo/production/services/data/ProcessService.java

```diff
@@ -2017,10 +2016,15 @@ private List<DocketData> getDocketData(List<Process> processes) {
      *            The process to create the docket data for.
      * @return The DocketData for the process.
      */
-    private DocketData getDocketData(Process process) {
+    private static DocketData getDocketData(Process process) throws IOException {
         DocketData docketdata = new DocketData();
 
         docketdata.setCreationDate(process.getCreationDate().toString());
+        URI metadataFilePath = fileService.getMetadataFilePath(process);
+        docketdata.setMetadataFile(fileService.getFile(metadataFilePath).toString());
```

Comment:
In my opinion the cast `toString` should not be used to get the path to a file - there are better and more reliable methods to get this information.

Reply:
Changed